examples/fluent-bit/oomkill-prevention/README.md

```diff
@@ -193,6 +193,20 @@ A couple of key notes:
 
 When the `storage.total_limit_size` is reached, the oldest chunks will be deleted first to make room for new chunks. *Older buffered logs will be lost in favor of making room for newer logs.*
 
+In AWS for Fluent Bit 2.X.Y+ there is an info level message logged when this limit is reached and chunks will be removed:
```

Comment:
added here: https://github.com/aws/aws-for-fluent-bit/pull/590/files

will be released later

Reply:
https://github.com/aws/aws-for-fluent-bit/releases/tag/v2.31.9